src/components/App.jsx

```diff
@@ -0,0 +1,20 @@
+import { Routes, Route } from 'react-router-dom';
+import Books from '../routes/Books';
+import Authors from '../routes/Authors';
+import NotMatch from '../routes/NotMatch';
+import Layout from './Layout';
+import '../styles/App.css';
+
+function App() {
+  return (
+    <Routes>
+      <Route path="/" element={<Layout />}>
+        <Route path="books" element={<Books />} />
+        <Route path="authors" element={<Authors />} />
+        <Route path="*" element={<NotMatch />} />
```

Comment:
- Great Job adding routes to your books page 👍🏽 
- It will be better if we replaced authors' routes with categories.